release.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
I assume this can go in the frontend config?

Reply:
Maybe, but I'm not sure if it's worth it. The config used by gundam has little in common with this one, because we use the angular preset, npm publish with semantic-release, and configure `@semantic-release/github` to point to GHE